server/backend/quiz/views.py

```
@@ -0,0 +1,118 @@
+    queryset = Category.objects.all()
+    serializer_class = CategorySerializer
+
+class QuizAPIView(APIView):
+    def get(self, request, category_id):
+        try:
+            # Ensure category_id is valid
+            category_id = int(category_id)
+        except (TypeError, ValueError):
+            return Response(
+                {'error': 'Invalid category ID'}, 
+                status=status.HTTP_400_BAD_REQUEST
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)